client/src/app/components/gui-editor/sidebar/editor-sidebar.component.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    Component,
+    style,
+    animate,
+    state,
+    transition,
+    trigger,
+    EventEmitter,
+    Output
+} from "@angular/core";
+import {NgFor} from "@angular/common";
+import {GuiEditorService} from "../gui-editor.service";
+import {VisibilityState} from "../animation.states";
+import {GuiEditorEvent, GuiEditorEventType, ShowSidebarEvent} from "../gui-editor.events";
```

Comment:
Remove unused imports. You can do `Optimize imports` in IntelliJ Idea by pressing  `⌃` + `⌥` + `O`, it'll remove imports that aren't being used or aren't valid.